pkg/workload/workload.go

```diff
@@ -282,6 +282,27 @@ func dropExcludedResources(input corev1.ResourceList, excludedPrefixes []string)
 	return res
 }
 
+func (i *Info) LqUsage(ctx context.Context, c client.Client, resWeights map[corev1.ResourceName]float64) (float64, error) {
```

Comment:
```suggestion
func (i *Info) LocalQueueUsage(ctx context.Context, c client.Client, resWeights map[corev1.ResourceName]float64) (float64, error) {
```

This is the wide-scope function. So let's use fully sentence.

Reply:
Done